packages/genesys-spark-components/src/components/beta/gux-month-picker/gux-month-calendar/gux-month-calendar.tsx

```diff
@@ -66,6 +67,18 @@ export class GuxMonthCalendar {
   @State()
   locale: string;
 
+  @State()
+  previewValue: string;
+
+  /**
+   * Controls hiding and showing the month-calendar
+   */
+  @Prop({ mutable: true })
+  expanded: boolean = false;
```

Comment:
@katie-bobbe-genesys Thoughts on my implementation of closing the calendar on focus out? I realize line 77 introduces a new prop called `expanded` and without the client using this the calendar will not open, which would be a breaking change. I'm trying to figure out a way to toggle visibility of the calendar without adding a new prop.